erpnext-openmrs/src/main/java/com/ozonehis/eip/erpnext/openmrs/processors/ServiceRequestProcessor.java

```diff
@@ -73,16 +73,16 @@ public void process(Exchange exchange) {
             if (patient == null || encounter == null || serviceRequest == null) {
                 throw new IllegalArgumentException("Patient, Encounter or ServiceRequest not found in the bundle");
             } else {
-                if (serviceRequest.getStatus().equals(ServiceRequest.ServiceRequestStatus.ACTIVE)
-                        && serviceRequest.getIntent().equals(ServiceRequest.ServiceRequestIntent.ORDER)) {
-                    log.info("Processing ServiceRequest for patient with UUID {}", patient.getIdPart());
-                    var customer = customerMapper.toERPNext(patient);
-                    String eventType = exchange.getMessage().getHeader(Constants.HEADER_FHIR_EVENT_TYPE, String.class);
-                    if (eventType == null) {
-                        throw new IllegalArgumentException("Event type not found in the exchange headers");
-                    }
-                    if ("c".equals(eventType) || "u".equals(eventType)) {
-                        customerHandler.ensureCustomerExistsAndUpdate(producerTemplate, patient);
+                log.info("Processing ServiceRequest for patient with UUID {}", patient.getIdPart());
+                String eventType = exchange.getMessage().getHeader(Constants.HEADER_FHIR_EVENT_TYPE, String.class);
+                if (eventType == null) {
+                    throw new IllegalArgumentException("Event type not found in the exchange headers");
+                }
+                if ("c".equals(eventType) || "u".equals(eventType)) {
+                    if (serviceRequest.getStatus().equals(ServiceRequest.ServiceRequestStatus.ACTIVE)
```

Comment:
Nit
I generally prefer it other way round `eventType.equals("c")`

Reply:
I like to do it this way to avoid potential `NullPointerException` if `eventType` is `null`, although I'm already checking for null.